haystack/nodes/file_converter/pdf.py

```diff
@@ -15,9 +26,11 @@ def __init__(
         remove_numeric_tables: bool = False,
         valid_languages: Optional[List[str]] = None,
         id_hash_keys: Optional[List[str]] = None,
-        encoding: Optional[str] = "UTF-8",
-        keep_physical_layout: bool = False,
-    ):
+        encoding: Optional[str] = None,
+        keep_physical_layout: Optional[bool] = None,
+        sort_by_position: Optional[bool] = None,
```

Comment:
Let's not make this optional here:
```suggestion
        sort_by_position: bool = False,
```